cpp/src/cider/benchmarks/HashMapBenchmark.cpp

```diff
@@ -113,11 +113,11 @@ static std::tuple<std::vector<T>, std::vector<bool>> generateAndFillVector(
 
 template <typename KeyType, typename HashTableType>
 static void BM_Lookup(benchmark::State& state) {
-  size_t row_num = 100;
+  size_t row_num = state.range(0);
   KeyType value_min = std::numeric_limits<KeyType>::min();
   KeyType value_max = std::numeric_limits<KeyType>::max();
   size_t null_chance = 0;
-  GeneratePattern pattern = GeneratePattern::Random;
+  GeneratePattern pattern = GeneratePattern::Sequence;

```

Comment:
why change to sequence?

Reply:
We can find `collision` which is almost a fixed value, to better illustrate the performance of hashtables.